packages/router/test/adapters/contract/contract.spec.ts

```diff
@@ -31,12 +31,21 @@ import { createStubInstance, SinonStubbedInstance, stub, SinonStub, restore, res
 import { TransactionManagerInterface } from "@connext/nxtp-contracts/typechain/TransactionManager";
 import { RouterInterface } from "@connext/nxtp-contracts/typechain/Router";
 import { routerAddrMock, routerContractAddressMock } from "../../utils";
-import { messagingMock, signerAddress, txServiceMock } from "../../globalTestHook";
+import { routerAddress, messagingMock, signerAddress, txServiceMock } from "../../globalTestHook";
 import { SanitationCheckFailed } from "../../../src/lib/errors";
 import { ERC20Interface } from "@connext/nxtp-contracts/typechain/ERC20";
 
 const requestContext = createRequestContext("TEST");
 const encodedDataMock = "0xabcde";
+const chainIdMock = txDataMock.sendingChainId;
+const onchainTxMock = {
+  to: routerContractAddressMock,
+  data: encodedDataMock,
+  value: constants.Zero,
+  chainId: chainIdMock,
+  from: routerAddress,
+};
```

Comment:
This should line up with the `txReceiptMock` as well

Reply:
Yeah a good point. We should def condense some of the mocking we've done across all unit tests in router at some point